doc/classes/CharacterBody2D.xml

```diff
@@ -192,12 +189,15 @@
 		<member name="up_direction" type="Vector2" setter="set_up_direction" getter="get_up_direction" default="Vector2(0, -1)">
 			Direction vector used to determine what is a wall and what is a floor (or a ceiling), rather than a wall, when calling [method move_and_slide]. Defaults to [code]Vector2.UP[/code]. If set to [code]Vector2(0, 0)[/code], everything is considered a wall. This is useful for topdown games.
 		</member>
+		<member name="wall_min_slide_angle" type="float" setter="set_wall_min_slide_angle" getter="get_wall_min_slide_angle" default="0.261799">
+			Minimum angle (in radians) where the body is allowed to slide when it encounters a slope. The default value equals 15 degrees.
```

Comment:
Maybe add something like this? Also to 3D I suppose.
```suggestion
			Minimum angle (in radians) where the body is allowed to slide when it encounters a slope. The default value equals 15 degrees.
			This property only affects movement when [member motion_mode] is [constant MOTION_MODE_FLOATING].
```

Reply:
Yes good catch! It's a bit different for 3D and there already was a message and change few words to harmonise the text.